packages/payload/src/queues/operations/runJobs/index.ts

```diff
@@ -253,11 +263,19 @@ export const runJobs = async ({
 
   if (jobsToDelete && jobsToDelete.length > 0) {
     try {
-      await req.payload.delete({
-        collection: jobsCollectionSlug,
-        req,
-        where: { id: { in: jobsToDelete } },
-      })
+      if (req.payload.config.jobs.runHooks) {
+        await req.payload.delete({
+          collection: jobsCollectionSlug,
+          depth: 0, // can be 0 since we're not returning anything
+          disableTransaction: true,
+          where: { id: { in: jobsToDelete } },
+        })
+      } else {
+        await req.payload.db.deleteMany({
+          collection: jobsCollectionSlug,
```

Comment:
We should add `returning: false` to db.deleteMany`. Doesn't need to be in this PR.

Reply:
That's default! deleteMany doesn't return anything